soc/xtensa/intel_adsp/common/soc_mp.c

```diff
@@ -354,27 +368,18 @@ void arch_start_cpu(int cpu_num, k_thread_stack_t *stack, int sz,
 	uint32_t ietc = IDC_MSG_POWER_UP_EXT((long) z_soc_mp_asm_entry);
 
 	IDC[curr_cpu].core[cpu_num].ietc = ietc;
-	IDC[curr_cpu].core[cpu_num].itc = IDC_MSG_POWER_UP | IPC_IDCITC_BUSY;
+	IDC[curr_cpu].core[cpu_num].itc = IDC_MSG_POWER_UP | BIT(31);
```

Comment:
not a logic change, but why? Even if it's the same value on all platforms, "BUSY" tells me much more than 31.

Reply:
Looks like a mis-squash, will fix.